<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QWidget" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>1891</width>
     <height>891</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="labelshow_camera">
      <property name="minimumSize">
       <size>
        <width>1024</width>
        <height>768</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1024</width>
        <height>768</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>911</y>
     <width>1901</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="light_btn">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#select_btn:focus{border-radius:10px;background-color: rgb(255, 255, 255);border: 1px solid #E8E8E8;color: rgb(45, 60, 99);}
QPushButton#select_btn{border-radius:10px;background-color: rgb(255, 255, 255);border: 1px solid #E8E8E8;color: rgb(45, 60, 99);}</string>
      </property>
      <property name="text">
       <string>打开警示灯</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="buzzer_btn">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#select_btn:focus{border-radius:10px;background-color: rgb(255, 255, 255);border: 1px solid #E8E8E8;color: rgb(45, 60, 99)}
QPushButton#select_btn{border-radius:10px;background-color: rgb(255, 255, 255);border: 1px solid #E8E8E8;color: rgb(45, 60, 99)}</string>
      </property>
      <property name="text">
       <string>打开蜂鸣器</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
